### DISCUSSION TITLE
weeklies/7

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#127] [Comments]
https://hutusi.com/articles/weekly-7 

 本文为周刊第七期，前期的周刊参见：周刊系列。

**Comment by T-Tenma** [comment]
学习如何学习收益颇多

**Comment by Mayandev** [comment]
No more Google give us more alternative，but we can't refuse the accessibility that Google enviroment gave us.

**Comment by hutusi** [comment]
@T-Tenma 标题感觉读起来有点别扭，我又改成了“如何学习”

**Comment by hutusi** [comment]
@Mayandev we can!  😝

**Comment by petite-dd** [comment]
我认为自己学习能力还可以，总结下来应该是：尽可能少的记忆，尽可能快的通读，尽可能早的上手。所以一般我要用个新语言，约莫2天就能开始项目了😂，但肯定和作者这样优秀的互联网深耕者的学习仔细程度是有区别的。
GOOGLE的logo变得更扁平，更符号化，让一切变得简单这或许也是互联网的思想和表达方式吧。
看到核能那一块想到了三体中能源无限的时代，那必定是一次巨大的革命。

**Comment by geektutu** [comment]
看到了我的专题系列，感谢博主推荐~
